From 203b0a64f979ab03e8b009767938b61d45236ec0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Thu, 17 Oct 2024 14:04:24 +0200
Subject: [PATCH] build: Don't override install mode for the binary

Files in bindir need the marked as executable, otherwise they cannot be executed.
---
 src/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/meson.build b/src/meson.build
index 815e5dd..3f9913b 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -35,7 +35,6 @@ configure_file(
   configuration: conf,
   install: true,
   install_dir: get_option('bindir'),
-  install_mode: 'r-xr--r--'
 )
 
 binary_sources = [
